metricbeat/module/docker/container/data.go

```diff
@@ -17,24 +16,45 @@ func eventsMapping(containersList []dc.APIContainers) []common.MapStr {
 	}
 	return myEvents
 }
-func eventMapping(mycontainer *dc.APIContainers) common.MapStr {
+
+func eventMapping(cont *dc.APIContainers) common.MapStr {
 
 	event := common.MapStr{
-		"@timestamp": time.Now(),
+		"timestamp": time.Now(),
```

Comment:
@tsg Agree. Before one was overwriting the other (or at least it tried and didn't as it was not common.Time). With the above implementation each event can have its separate timestamp instead of one for all events of this batch. But as the heavy work happens when the info from the docker api is fetched and the timestamp is generated afterwards, I suggest to just remove it.

Perhaps @douaejeouit can share some insights here?